### DISCUSSION TITLE
[委托] 急急急我朋友在零号空洞枯败花圃以太活性11第二层走失了（已解决）

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#1243] [其他]
![微信图片_20240811102921](https://github.com/user-attachments/assets/1453a6b6-6d01-4d8f-8be8-811a53198722)

如题，找到我朋友报酬150菲林，15000丁尼，没找到报酬50菲林，20000丁尼

**Comment by dveam** [comment]
兄弟，多关心一下你自己吧
![IMG_20240811_103237](https://github.com/user-attachments/assets/0803b078-dcc6-461d-bf37-060efd07c078)

**Reply by Soldier11-ObsidianBarracks** [reply]
没事现在不是了，而且击毙的是安比关我11号什么事（

**Comment by ZhuYuan-148** [comment]
怎么会有人跑到那么危险的地方？治安局这边会尽快找到空洞负责人出发寻找。

**Reply by Soldier11-ObsidianBarracks** [reply]
感谢朱鸢探长

**Reply by talataos** [reply]
原来治安官已经渗透绳网了

**Comment by morramu** [comment]
这个委托我接了

**Comment by Elysia-Ego-093** [comment]
这个委托我接了，艾莲30分钟后就来接你
(本绳匠用艾莲全程单通烈度11，技术绝对可信)

**Reply by Soldier11-ObsidianBarracks** [reply]
嫩强啊

**Comment by zzz14121** [comment]
这个委托我接了 我把尼尼微吸走

-

**Comment by Phaethon10031** [comment]
这个委托我接了！

**Comment by qbsozo** [comment]
这个委托我接了

**Comment by luo-luo-o** [comment]
这个委托我接了！

**Comment by XxnjiIAJ** [comment]
这个委托我接了

**Comment by gaojiaobei** [comment]
我有相关委托的经验！
